src/main/scala/org/broadinstitute/dsde/workbench/sam/dataAccess/LdapRegistrationDAO.scala

```diff
@@ -97,7 +97,7 @@ class LdapRegistrationDAO(
   override def disableAllHumanIdentities(samRequestContext: SamRequestContext): IO[Unit] = {
     //The iam.gserviceaccount.com filter is in place to ensure that only human identities are disabled. Service Accounts (both regular SAs and Pet SAs) are
     // currently exempt for ToS-enforcement, thus, they're ignored when disabling identities
-    val humanIdentityDnsIO = (executeLdap(IO(ldapConnectionPool.search(peopleOu, SearchScope.SUB, "(!(mail=*.iam.gserviceaccount.com))")), "getAllIdentitiesToDisable", samRequestContext) map { results =>
+    val humanIdentityDnsIO = (executeLdap(IO(ldapConnectionPool.search(peopleOu, SearchScope.SUB, "(!(mail=*.gserviceaccount.com))")), "getAllIdentitiesToDisable", samRequestContext) map { results =>
```

Comment:
Most SAs end in `iam.gserviceaccount`, but there are some that don't. This ensures that we don't accidentally remove them from the enabledUsers group